### ISSUE TITLE
Timed out while waiting for the machine to boot

### ISSUE THREAD

**Issue by juliuscaesarnl** [laravel/homestead#1607]
## Versions
  - Vagrant: 2.2.14
  - Provider: Virtualbox 6.1.16
  - Homestead: 11.4.0
  - Box (settler): 10.1.1

## Host operating system
Windows 10 build 19042.746

## Homestead.yaml
```yaml
ip: 192.168.10.10
memory: 2048
cpus: 2
provider: virtualbox
authorize: ~/.ssh/id_rsa.pub
keys:
  - ~/.ssh/id_rsa
folders:
  - map: 'C:\projects\portal'
    to: /home/vagrant/code
sites:
  - map: portal.test
    to: /home/vagrant/code/public
databases:
  - portal
features:
  - mysql: true
  - mariadb: false
  - ohmyzsh: false
  - webdriver: false
backup: false
name: portal
hostname: portal
```

## Vagrant destroy & up output
[Vagrant output](https://gist.github.com/juliuscaesarnl/a8c933b825c5165185dab259fbf5c9f8)
Note: creation of VM and first time booting always succeed.

## Expected behavior
The VM boots up without timeouts.

## Actual behavior
Almost every time, the 'vagrant up' command times out. It seems that there's some process inside the VM which is stalling the CPU on boot. What I've noticed is that booting continues as soon as I open the display output of the VM from VirtualBox. It also shows in the display output that the CPU is stalling. See the screenshot:
![image](https://user-images.githubusercontent.com/39700840/106621662-100f1380-6573-11eb-89a8-eca415f78e39.png)

Here's what I've tried so far:
- Use different versions of Vagrant down to 2.2.10
- Use different VirtualBox versions down to 6.1.12
- Clean install both Vagrant and Virtualbox
- Different versions of Homestead and Settler

On top of this, the rest of my team is also experiencing the same issue, so it's not isolated to my machine.
The issue seems to be introduced since Homestead v11.x and Settler v10.x

## Steps to reproduce
1. Execute 'vagrant up'
2. Wait until the VM boots up

## References
None

**Comment by svpernova09** [comment]
Do you have virtualization enabled in your BIOS? This is a Virtualbox failure, nothing Homestead can do if the provider can't boot the machine. Does the same thing happen with another vagrant box? I'd assume so.

**Comment by juliuscaesarnl** [comment]
I do have it enabled, but I doubt that this has something to do with this issue. I've had it enabled for the last 2 years and been using Homestead and VirtualBox since then, without any issues.

Either way, I've booted up a different box (ubuntu/focal64) with a basic Vagrantfile and I get the same behavior, so it's definitely not a Homestead issue. Thanks for the suggestion! Will close the issue now.

**Comment by svpernova09** [comment]
If you need help chasing down ideas or just want to ask general virtualization questions I'm in #homestead on the Laravel Discord Server. There are a few people around that might can help.

**Comment by juliuscaesarnl** [comment]
Update: after trying different things I finally figured out the solution for my machine.
Your clue about virtualization gave me inspiration to look in that direction. Found the solution on [StackOverflow](https://stackoverflow.com/a/65849976/6681778).

Basically, I found out that CPUs stalling in a VM is related to nested virtualization and that enabling this option should fix the issue. Unfortunately it was grayed out. I once enabled the Hyper-V feature, then disabled it again but apparently this doesn't fully remove/disable the feature. So this was the reason I couldn't enable the Nested VT-x option in the VM's settings.
![image](https://user-images.githubusercontent.com/39700840/106906796-d3bbee80-66fd-11eb-988b-9e65652599c7.png)

I already knew that Hyper-V and other virtualization software don't play so well together, but I didn't figure that this could be the issue since I disabled it (guess not). Either way, I can finally boot up the VM after months of trying different solutions.
